code/modules/mob/new_player/new_player.dm

```diff
@@ -169,6 +169,13 @@
 		ViewManifest()
 
 	if(href_list["SelectedJob"])
+	
+		//Prevents people rejoining as same character.

```

Comment:
i know it isn't virgo's fault but jesus christ reading this snippet of code has hurt me inside